src/AaveV2FreezeReserves_20230619/V2-to-V3-MIGRATION-NEXT-STEPS.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Chaos Labs V2 to V3 Migration Next Steps
+author: Chaos Labs (@Chaos Labs)
+shortDescription: Chaos Labs V2 to V3 Migration Next Steps
+discussions: https://governance.aave.com/t/arfc-chaos-labs-v2-to-v3-migration-next-steps/13701
+created: 2023-06-19
+---
+
+## Simple Summary
+
+This AIP proposes to Freeze the the 1INCH, ENS, LINK, MKR, SNX & UNI reserves on the Aave V2 Ethereum pool
+
+## Motivation
+
+Following the successful deployment of Aave V3 on the Ethereum network, we propose a reassessment of smaller-to-medium capitalization tokens. Unlike CRV (we will soon contribute further insights on this topic, as the complexity of the CRV market, primarily due to its substantial scale, warrants a more nuanced examination), these markets exhibit a more modest size and generally can be conveniently migrated. This diverges from preceding deliberations on market freezes prior to the V3 launch. With V3 live, invoking a freeze on V2 markets does not preclude users from capitalizing on these markets. Instead, it enhances the community’s capacity to safeguard user funds more effectively, as V3’s advanced risk management mechanisms provide a more secure environment. This proactive strategy aims to enforce a freeze on low-to-medium capital assets on V2, thereby averting a scenario resembling Aave V2’s current overexposure to CRV while permitting users to engage with Aave via V3 markets.
+## Specification
```

Comment:
```suggestion

## Specification
```

If you save, your editor should automatically lint that file. Doesn't this work for you?